bundles/org.openhab.binding.hydrawise/src/main/java/org/openhab/binding/hydrawise/internal/api/model/BocTopologyActual.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hydrawise.internal.api.model;
+
+import java.util.List;
+
+/**
+ *
```

Comment:
Our guidelines state that for every class the class-level docblock describes its goal/meaning.

Reply:
Check others

Reply:
I believe that these classes would even be better of without getters and setters (and thus public fields). Given they are just json-dto's that would save a lot of code and function the same.